eodhp-web-presence/templates/web-presence-deploy.yaml

```diff
@@ -42,3 +42,7 @@ spec:
               value: {{ .Values.catalogue_data.url | default "https://my-stac-browser-data.com/catalog.json" }}
             - name: NOTEBOOKS_URL
               value: {{ .Values.notebooks.url | default "https://notebooks.co.uk/apphub" }}
+            - name: ALLOWED_HOSTS
+              value: {{ .Values.hosting.allowed_hosts | default "localhost,127.0.0.1,my.website.co.uk" }}
```

Comment:
I wouldn't include my.website.co.uk here as an example, localhost,127.0.0.1 is sufficient as these may actually be useful in dev.

Reply:
Updated